### ISSUE TITLE
Make it possible to configure unlimited reconnects / retries

### ISSUE THREAD

**Issue by PatrickHuetter** [vert-x3/vertx-rabbitmq-client#22]
At this time it's only possible to define a maximum of retries for a reconnect. There should be also the option to configure unlimited retries (e.g. with -1) or similar.

**Comment by poiuytrez** [comment]
You are right. The workaround is to set a high value Integer.MAX_VALUE for example.
